services/retrieve_and_rank/v1.js

```diff
@@ -54,10 +54,16 @@ module.exports = function (RED) {
           };
 
           if (config.rankername) {
-            params.training_metadata = '{\"name\": \"'+config.rankername+'\"}';
+            params.training_metadata = "{\"name\": \""+config.rankername+"\"}";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2405715173/issues/source?bid=3225115&fileBranchId=3233651#l57)